src/backend/distributed/metadata/node_metadata.c

```diff
@@ -757,6 +757,14 @@ master_update_node(PG_FUNCTION_ARGS)
 		if (force)
 		{
 			handle = StartLockAcquireHelperBackgroundWorker(MyProcPid, lock_cooldown);
+			if (!handle)
+			{
+				StringInfo lockCooldownStringValue = makeStringInfo();
+				appendStringInfo(lockCooldownStringValue, "\"%d\"", lock_cooldown);
```

Comment:
are the quotes required? (fine if they are)

Reply:
I think we should use a utility method to convert an int to a string. In `set_config_option` we can directly call the utility method with `lock_cooldown` argument.